plugins/slack_webhooks/test/samson_slack_webhooks/samson_plugin_test.rb

```diff
@@ -14,10 +14,21 @@
       Samson::Hooks.fire(:buddy_request, deploy)
     end
 
-    it "does not send notifications when disabled" do
-      webhook.buddy_request = false
-      SlackWebhookNotification.any_instance.expects(:deliver).never
-      Samson::Hooks.fire(:buddy_request, deploy)
+    describe "when disabled" do
+      before { webhook.buddy_request = false }
+
+      it "does not send notifications when disabled" do
+        SlackWebhookNotification.any_instance.expects(:deliver).never
+        Samson::Hooks.fire(:buddy_request, deploy)
+      end
+
+      # using full http test here to make sure our hackery works
+      it "sends global hook when configured" do
+        with_env SLACK_GLOBAL_BUDDY_REQUEST: 'http://foo.com/baz#bar' do
+          stub_request(:post, "http://foo.com/baz")
+          Samson::Hooks.fire(:buddy_request, deploy)
```

Comment:
Warning: Above mocha line was completely made up as I didn't look up the syntax! :)